src/test/java/org/cryptomator/cryptofs/CryptoFileChannelWriteReadIntegrationTest.java

```diff
@@ -632,6 +632,7 @@ public void testConcurrentWriteAndTruncate() throws IOException {
 						buf.flip();
 					}
 				});
+				Thread.sleep(1000);

```

Comment:
To be removed for CI, rather use a barrier, countdownlatch, etc.

Reply:
Done in https://github.com/cryptomator/cryptofs/pull/270/commits/9d93701d58d827aeb5dfbe34148e8e75cfdf37b6

Reply:
for the record, I meant like in 5363d4b